src/Service.Tests/Unittests/SqlMetadataProviderUnitTests.cs

```diff
@@ -105,7 +106,11 @@ public void CheckTablePrefix(string databaseName, string schemaName, string expe
             Mock<IQueryExecutor> queryExecutor = new();
             IQueryBuilder queryBuilder = new MsSqlQueryBuilder();
 
-            SqlMetadataProvider<SqlConnection, SqlDataAdapter, SqlCommand> provider = new MsSqlMetadataProvider(runtimeConfigProvider, queryExecutor.Object, queryBuilder, sqlMetadataLogger);
+            Mock<IAbstractQueryManagerFactory> engineFactory = new();
+            engineFactory.Setup(x => x.GetQueryBuilder(It.IsAny<DatabaseType>())).Returns(queryBuilder);
```

Comment:
nit: this should be named as managerFactory

Reply:
fixed.